packages/core/src/Stores/client-store.js

```diff
@@ -2133,17 +2108,8 @@ export default class ClientStore extends BaseStore {
         const is_client_logging_in = login_new_user ? login_new_user.token1 : obj_params.token1;
 
         if (is_client_logging_in) {
-            const redirect_url = sessionStorage.getItem('redirect_url');
-            const local_pre_appstore = localStorage.getItem('is_pre_appstore');
-            if (
-                local_pre_appstore === 'true' &&
-                redirect_url?.endsWith('/') &&
-                (isTestLink() || isProduction() || isLocal() || isStaging())
-            ) {
-                window.history.replaceState({}, document.title, '/appstore/traders-hub');
-            } else {
-                window.history.replaceState({}, document.title, sessionStorage.getItem('redirect_url'));
-            }
+            window.history.replaceState({}, document.title, '/appstore/traders-hub');
+
```

Comment:
we still need to redirect user to the according `redirect_url` ,please don't remove this part
 ```suggestion
const redirect_url = sessionStorage.getItem('redirect_url');
            if (
                redirect_url?.endsWith('/') &&
                (isTestLink() || isProduction() || isLocal() || isStaging())
            ) {
                window.history.replaceState({}, document.title, '/appstore/traders-hub');
            } else {
                window.history.replaceState({}, document.title, sessionStorage.getItem('redirect_url'));
```